stripe/src/main/java/com/stripe/android/Stripe.java

```diff
@@ -246,6 +314,61 @@ public void createToken(
     }
 
     /**
+     * Blocking method to create a {@link Source} object using this object's
+     * {@link Stripe#mDefaultPublishableKey key}.
+     *
+     * Do not call this on the UI thread or your app will crash.
+     *
+     * @param params a set of {@link SourceParams} with which to create the source
+     * @return a {@link Source}, or {@code null} if a problem occurred
+     *
+     * @throws AuthenticationException failure to properly authenticate yourself (check your key)
+     * @throws InvalidRequestException your request has invalid parameters
+     * @throws APIConnectionException failure to connect to Stripe's API
+     * @throws CardException the card cannot be charged for some reason
```

Comment:
It can be thrown, technically. I can always just go to the superclass, and call it a StripeException, though that would consume the lower ones. Do we have source-specific ones?